Formula/yarn.rb

```diff
@@ -17,7 +15,7 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    cp_r '.', prefix
```

Comment:
Yes, I checked in locally.

- `brew remove yarn`
- `brew install yarn -s`
- `/usr/local/bin/yarn`

<img width="235" alt="screenshot 2016-11-15 20 44 14" src="https://cloud.githubusercontent.com/assets/208544/20309987/5de23eec-ab74-11e6-84cd-331d603fc05b.png">